src/components/course-listing.cjsx

```diff
@@ -60,12 +60,19 @@ CourseListing = React.createClass
       isStudent = _.find roles, (role) -> role.type is 'student'
       isTeacher = _.find roles, (role) -> role.type is 'teacher'
 
+      courseType = if isConceptCoach then 'Concept Coach' else 'Tutor'
+
+      if courselink?
+        courseTypeFlag = "(#{courseType})"
+      else
+        courseTypeFlag = <div className='course-type-flag'>{courseType}</div>
+
```

Comment:
Yeah, I suspect it's old code as well.  git blame suggests that @pandafulmanda added it  last summer.  Why don't you take out the courseLink conditionals and see how it works.